src/native/managed/cdacreader/src/Legacy/SOSDacImpl.cs

```diff
@@ -512,7 +512,22 @@ public unsafe int GetObjectStringData(ulong obj, uint count, char* stringData, u
     }
     public unsafe int GetOOMData(ulong oomAddr, void* data) => HResults.E_NOTIMPL;
     public unsafe int GetOOMStaticData(void* data) => HResults.E_NOTIMPL;
-    public unsafe int GetPEFileBase(ulong addr, ulong* peBase) => HResults.E_NOTIMPL;
+
+    public unsafe int GetPEFileBase(ulong addr, ulong* peBase)
+    {
+        try
+        {
+            Contracts.ILoader contract = _target.Contracts.Loader;
+            Contracts.ModuleHandle handle = contract.GetModuleHandle(addr);
+            *peBase = contract.GetILBase(handle);
```

Comment:
don't you need to check the `IsReflectionEmit` flag the same way that `GetPEFileBaseImpl` does?